src/modules/navigator/mission.cpp

```diff
@@ -582,11 +582,14 @@ Mission::set_mission_items()
 
 	/* mission item that comes after current if available */
 	struct mission_item_s mission_item_next_position;
+	struct mission_item_s mission_item_next_next_position;
```

Comment:
What? That's a confusing name :smile:. What about `next but one` or `after next`?